algorithm/cgal/cgal.cc

```diff
@@ -2119,6 +2079,198 @@ int ClipSurfaceMeshes(size_t target_count, emscripten::val getTargetMesh,
   return STATUS_OK;
 }
 
+int ParallelCutSurfaceMeshes(size_t target_count, emscripten::val getTargetMesh,
```

Comment:
Right now this one isn't used right? But we can switch to it once cgal becomes thread safe?

Reply:
Actually, this is the one in use at the moment -- it's just compiled with threading off.

We should see if there's a performance difference between the two algorithms.